thirdparty/gravity-forms.cls.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * The Third Party integration with Gravity Forms.
+ *
+ * @since       4.1.0
+ */
+namespace LiteSpeed\Thirdparty;
+
+defined( 'WPINC' ) || exit;
+
+class Gravity_Forms
+{
+    /**
+     * Check if GF is enabled and disable LSCWP on gf-download and gf-signature URI
+     *
+     * @since 4.1.0 #900899 #827184
+     */
+    public static function preload()
```

Comment:
Try to rename this function to `detect` and drop the `add_action` above to see if it can work or not.